robolectric/src/main/java/org/robolectric/internal/dependency/CompositeDependencyResolver.java

```diff
@@ -0,0 +1,41 @@
+package org.robolectric.internal.dependency;
+
+import java.net.URL;
+import java.util.List;
+
+public class CompositeDependencyResolver implements DependencyResolver {
+  private List<DependencyResolver> dependencyResolvers;
+
+  public CompositeDependencyResolver() {
```

Comment:
Does a default constructor make sense here? Especially since you can't set the dependencyResolvers later. I'd vote to remote it.